app/Models/UserSettings.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace Adshares\Adserver\Models;
+
+use Adshares\Adserver\Models\Contracts\Camelizable;
+use Adshares\Adserver\Models\Traits\AutomateMutators;
+use Adshares\Adserver\Models\Traits\Serialize;
+use Adshares\Adserver\Models\Traits\ToArrayCamelize;
+use Illuminate\Database\Eloquent\Model;
+
+class UserSettings extends Model implements Camelizable
+{
+    use AutomateMutators;
+    use Serialize;
+    use ToArrayCamelize;
+
+    public static $default_notifications = [
+        'billing' => ['email' => true, 'notify' => true],
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "notify" 5 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=php%3AS1192)
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "email" 5 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=php%3AS1192)